runners/gradle-plugin/gradle.properties

```diff
@@ -1 +0,0 @@
-kotlin.stdlib.default.dependency=false
```

Comment:
Is this not relevant anymore? It was a recent fix (#2543)

https://mbonnin.medium.com/the-different-kotlin-stdlibs-explained-83d7c6bf293

Reply:
I see. It'd be nice to have one less flag. The problem was somewhat difficult to track and test (#2546), so if you don't mind - let's restore it just to be on the safe side and so that it doesn't block this PR?

I've written it down to be investigated later, but if you want and have the time - it can be extracted into a separate PR, we'll need it anyway

Reply:
Also, the chance of accidentally including the wrong stdlib could be improved with included-builds, which is on the list #2652. The regular Dokka code could be built completely separately from the Gradle plugin. (It's not a critical requirement, just extra security)